projects/storefrontapp/src/app/spartacus/features/digital-payments-feature.module.ts

```diff
@@ -17,4 +16,40 @@ import {
     }),
   ],
 })
-export class DigitalPaymentsFeatureModule {}
+export class DigitalPaymentsFeatureModule {
+  static forB2C(): ModuleWithProviders<DigitalPaymentsFeatureModule> {
```

Comment:
another drawback is more code for customer if they decide to not use out feature modules

because if we put each 'wrapper module' within the dp-lib, they can always use that instead of us putting this b2b dp module in out shellapp

Reply:
I like this approach, but one drawback is that the unused static methods are not tree-shaken.
It's not adding much of bytes to the bundle, but we'll have weigh in the trade-offs